build/webpack.base.conf.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
   resolve: {
     extensions: [".js", ".vue", ".json"],
     alias: {
-      vue$: "vue/dist/vue.esm.js",
+      //vue$: "vue/dist/vue.esm.js",
```

Comment:
I couldn't find why this was here.

Reply:
Hrmm, not sure. I'm OK w/ deleting if it works without it.

Reply:
It was probably boilerplate from some sort of create-vue-app analog from the original proof of concept; it was in the initial commit as far as I can tell. It's been too long ago for me to remember why it was in there :P

Reply:
Thank you both for your input. I will let this commented out since everything seems to work without it.